api/tests/routes/pro/patch_pro_user_has_seen_rgs_test.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+import pcapi.core.users.factories as users_factories
+
+from tests.conftest import TestClient
+
+
+@pytest.mark.usefixtures("db_session")
+def test_mark_as_seen(app):
+    user = users_factories.UserFactory(hasSeenProRgs=False)
+
+    client = TestClient(app.test_client()).with_session_auth(user.email)
```

Comment:
Tu peux utiliser la fixture `client` (à mettre en argument de la fonction de test, à la place de `app`) et écrire simplement : 

```suggestion
    client = client.with_session_auth(user.email)
```

(idem dans l'autre test plus bas)